README contains wrong package name for `npm install`

Pretty simple, the `npm install` command is broken, it needs to be scoped to the `@adobe` NPM group.

### Expected Behaviour

Can copy / paste `npm i -s <package>` and it work.

### Actual Behaviour

NPM blows up saying there's no package with that name.

### Reproduce Scenario (including but not limited to)

Copy the install command and run it in an empty project.

#### Steps to Reproduce

See above.

#### Platform and Version

N/A

#### Sample Code that illustrates the problem

N/A

#### Logs taken while reproducing problem

```shell
$ npm install --save inquirer-table-checkbox
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/inquirer-table-checkbox - Not found
npm ERR! 404
npm ERR! 404  'inquirer-table-checkbox@*' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

--- a/README.md
+++ b/README.md
@@ -15,7 +15,7 @@
 ## Installation
 
 ```sh
-npm install --save inquirer-table-checkbox
+npm install --save @adobe/inquirer-table-checkbox
 ```
 
 ## Usage

